pkg/sfu/connectionquality/connectionstats.go

```diff
@@ -266,7 +269,7 @@ func (cs *ConnectionStats) updateScoreFromReceiverReport(at time.Time) (float32,
 	if streamingStartedAt.After(agg.StartTime) {
 		agg.StartTime = streamingStartedAt
 	}
-	return cs.updateScoreWithAggregate(agg, at), streams
+	return cs.updateScoreWithAggregate(agg, time.Time{}, at), streams
```

Comment:
not using last RTCP report for down stream tracks as the connection quality is already coupled to the RTCP Receiver Report.